src/Sylius/Bundle/AdminBundle/templates/product_review/form/sections/details/rating.html.twig

```diff
@@ -4,4 +4,6 @@
 
 {% form_theme form '@SyliusAdmin/product_review/form_theme.html.twig' %}
 
-{{ form_row(form) }}
+<div class="sylius-rating">
+    {{ form_row(form) }}
+</div>
```

Comment:
Since it seems that the `sylius-rating` class is necessary for proper styling, it should be extracted into a `_sylius_admin_product_review_rating_row` theme block